pkg/manager/member/tikv_failover_test.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package member
+
+import (
+	"fmt"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"time"
+
+	. "github.com/onsi/gomega"
```

Comment:
Add revive instruction for `dot-import`.
You can choose whatever you like to follow revive linter suggests.
`retool do revive -formatter friendly -config revive.toml pkg/...`